portfolio/src/main/webapp/style.css

```diff
@@ -146,7 +144,27 @@ h2{
     border-radius: 7px;
     font-weight:bold;
 }
-img{
-    width:400px;
-    display:block;
-}
\ No newline at end of file
+#contact-container{
```

Comment:
please alphabetize all the css declarations: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
feel free to ignore this comment based on Tyler's suggestion to use prettier instead.

Reply:
Ideally we'd alphabetize, but I don't think it adds much value here and it's just busy work we'd rather you not do. Also, within Google there is usually tooling that will do this type of tedious work for you.